conf.py

```diff
@@ -139,7 +139,15 @@
 
 NAVIGATION_LINKS = {
     DEFAULT_LANG: (
-
+        (
+            (
+                ("/pages/guide.md", "General Guide"),
+                ("/pages/pvt.md", "PvT"),
+                ("/pages/pvz.md", "PvZ"),
+                ("/pages/pvp.md", "PvP"),
```

Comment:
```suggestion
                ("/guide/", "General Guide"),
                ("/pvt/", "PvT"),
                ("/pvz/", "PvZ"),
                ("/pvp/", "PvP"),
```
This is sort of non-intuitive, but it's how I had to wire lines 266-269 for the About page to show up as the main page. We could change that, of course.

Reply:
Good to know, didn't actually know how it worked :P